src/v/raft/recovery_stm.cc

```diff
@@ -375,7 +365,7 @@ ss::future<> recovery_stm::handle_install_snapshot_reply(
     // snapshot received by the follower, continue with recovery
     (*meta)->match_index = _ptr->_last_snapshot_index;
     (*meta)->next_index = model::next_offset(_ptr->_last_snapshot_index);
-    (*meta)->last_sent_offset = _ptr->_last_snapshot_index;
+    (*meta)->in_flight_offset = _ptr->_last_snapshot_index;
```

Comment:
if this is in-flight offset, should we set it at the point we send the install_snapshot request, not when we handle the reply?